### PR TITLE
Adding a new make:auth:empty

### PR DESCRIPTION
**Pull request by weaverryan**
This is the first of a few authenticator commands that I want to add.

It's not SUPER helpful - just a blank authenticator. My hope is that the other `make:auth:` commands will be used more often anyways (e.g. `make:auth:form`).

Cheers!

### CODE REVIEW

File: src/Command/MakeAuthenticatorEmptyCommand.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ */
+final class MakeAuthenticatorEmptyCommand extends AbstractCommand
+{
+    protected static $defaultName = 'make:auth:empty';
+
+    public function configure()
+    {
+        $this
+            ->setDescription('Creates a new Doctrine entity class')
```
Comment (yceruto) [inline src/Command/MakeAuthenticatorEmptyCommand.php]: copy+paste :) should be updated

File: tests/Command/FunctionalTest.php [hunk 2]
```diff
@@ -183,6 +184,14 @@ public function getCommandTests()
             ]
         ];
 
+        $commands['auth_empty'] = [
+            new MakeAuthenticatorEmptyCommand($generator),
+            [
+                // class name
+                'AppCustomVoter'
```
Comment (yceruto) [inline tests/Command/FunctionalTest.php]: same here :) `AppCustomAuthenticator`?

**Comment by weaverryan** [comment]
Thanks @yceruto :)

**Comment by javiereguiluz** [comment]
I like this, but I don't like the `:empty` part of the name. The rest of the makers also generate empty things and they are not called `make:contorller:empty`, `make:command:empty`, etc.  What if we rename this to just `make:auth` but keep using the 3-part names for the other auths (e.g. `make:auth:form`)

**Pull request merged**